### ISSUE TITLE
Resource Card Description - Markdown not taken into account

### ISSUE THREAD

**Issue by petzlux** [opendatalu/gouvlu#72]
In the following example the resource description has the following markdown content:

Ce fichier au format KML est actualisé toutes les 5 minutes.

Pour accéder au fichier, on doit ajouter un HTTP HEADER nommé API-KEY:486ac6e4-93b8-4369-9c6a-28f7c4e1a81f

Un exemple avec l'outil Curl:

```
curl -H 'API-KEY:486ac6e4-93b8-4369-9c6a-28f7c4e1a81f' https://my.chargy.lu/b2bev-external-services/resources/kml >> chargy.kml
```
----

This is rendered correctly in the admin interface, but is rendered as plain text in the public resource card .

![screen shot 2018-02-28 at 15 11 16](https://user-images.githubusercontent.com/2540428/36792051-b75b4fb2-1c99-11e8-83c5-c1cb8ee311b2.png)

**Comment by petzlux** [comment]
Linked to https://github.com/opendatateam/udata/issues/1426

**Comment by noirbizarre** [comment]
Will be fixed in udata 1.4 by https://github.com/opendatateam/udata/pull/1547

**Comment by noirbizarre** [comment]
Fixed by opendatateam/udata#1547 (udata 1.4)
